### PR TITLE
Add unpkg and jsdelivr for vega-loader.

### PR DESCRIPTION
**Pull request by jheer**
Changes:

**vega-loader**

- Update `package.json` to include `unpkg` and `jsdelivr` properties.

### CODE REVIEW

File: packages/vega-loader/package.json [hunk 1]
```diff
@@ -20,6 +20,8 @@
   "author": "Jeffrey Heer (http://idl.cs.washington.edu)",
   "main": "build/vega-loader.node.js",
   "module": "index",
+  "unpkg": "build/vega-loader.min.js",
```
Comment (domoritz) [inline packages/vega-loader/package.json]: Alternatively, you can add `"browser": build/vega-loader.min.`

Comment (jheer) [reply packages/vega-loader/package.json]: We already use browser in a more nuanced way, and in addition (according to vega/dataflow-api#3) Observable / d3-require doesn't use the browser field.

Comment (domoritz) [reply packages/vega-loader/package.json]: It does but only after the other two. https://github.com/d3/d3-require/blob/master/src/index.js

Comment (mbostock) [reply packages/vega-loader/package.json]: Yep—to clarify, d3-require *will* use the `browser` entry if it’s a string, but not if it’s an object.

**Pull request merged**